### DISCUSSION TITLE
Three ideas for additional features

### DISCUSSION THREAD

**Discussion by 0yzmo** [Jeffser/Alpaca#389] [Ideas]
Hi! Thank you for a great app!  Got a couple of thoughts that could perhaps make Alpaca even better (I don't really know if these ideas/improvements are possible though):

1. Possibility to "add prompt" to downloaded models (and/or perhaps modelfiles).
2. An indicator if the model is loaded, and perhaps an indicator that shows if the loaded model runs on cpu, gpu or both.
3. Give a recommended max model size based on system information (for noobs like me).

**Comment by Jeffser** [comment]
Hi thanks for the ideas, you can kinda do the first one by going into the `+` icon on the model manager and selecting `Create from existing model` there you can add some different context to the model by just explaining in plain english what you want. For example `You are a development bot, you will help me code in Python` or something like that.

**Reply by 0yzmo** [reply]
Wow, nice - thank you. I didn't know about this function :)

**Reply by Jeffser** [reply]
yeah many people don't know that's a thing, I should probably find a way of making it more obvious lol
